update_ensembl_data.py

```diff
@@ -41,17 +53,31 @@ def write_genome_data(self, species):
     def write_ensembl_release(self):
         release = self.ensembl_client.get_release()
         print(f"Using Ensembl release {release}")
-        path = "stdpopsim/catalog/ensembl_info.py"
+        path = os.path.join("stdpopsim","catalog", "ensembl_info.py")
         with self.write(path) as f:
             print("release = ", release, file=f)
 
 
-def main():
+def main(args):
+    # Lists species and exits if user requested list of species
+    if args.list_species:
+        for species in stdpopsim.all_species():
+            print(species.id)
+        exit(0)
```

Comment:
Does this work? I think you need to call ``sys.exit(0)``, right?

Rather than yanking out of the interpreter directly here, it's a bit more pleasant to return here, and let control flow continue normally.